### PR TITLE
Validator staking page UI issues

### LINKED ISSUE #4749: Issues with validator monitor/profile pages and staking
### Actual behavior

- Last generated date shows `Invalid date` on the validator status popover in the validator monitor view
Inside round Outside round Sanctioned Latest stakes Watched

- Even after I have watched a validator, the validator does not appear on the watched validators tab under the validator monitor page

- Enter account password on the edit stake flow does not have a back button
Performance

- text color and styling for the status modal on edit stake transaction flow is not correct
Staking confirmation

- `Rank`, `Validator weight` and `staking slots available` columns are not populated properly on the stakes page. Also the WalletAddress component is wrongly styled
Pasted Graphic 31

### Steps to reproduce
- Visit the validator monitor page issues observed above would be seen
- Visit the a validator profile page, issues observed above would be seen
- initiate the add/edit stake flow on a validator the above issue would be observed

### Which version(s) does this affect? (Environment, OS, etc...)
al

### PR DESCRIPTION
**Pull request by oskarleonard**
### What was the problem?

This PR resolves #4749

### How was it solved?

- Adding dark mode state to text
- Fixed watch list for validators
- Renamed table headers in validator page
- Put password back button in correct place for staking flow

### How was it tested?

Manually

### CODE REVIEW

**Review by ManuGowda — COMMENTED** [review]
Screenshot 2023-03-28 at 10 58 42 AM

**Comment by ManuGowda** [comment]
Screenshot 2023-03-28 at 11 02 33 AM

File: src/modules/wallet/components/TokenCard/TokenCard.js [hunk 1]
```diff
@@ -26,15 +27,13 @@ const TokenCard = ({ lockedBalance, address, token }) => {
             value={convertFromBaseDenom(availableBalance, token)}
           />
         )}
-        {!lockedBalance ? null : (
-          <DialogLink
-            data-testid="locked-balance"
-            component="lockedBalance"
-            data={{ address }}
+        {lockedBalance > 0 && (
```
Comment (ManuGowda) [inline src/modules/wallet/components/TokenCard/TokenCard.js]: lockedBalance can be string? so its best to do `BigInt(lockedBalance) > BigInt(0)` for safer check?

File: src/modules/wallet/components/overview/overview.js [hunk 2]
```diff
@@ -78,17 +78,14 @@ const Overview = ({ isWalletRoute, history }) => {
...
 
-  const renderTokenCard = useCallback(
-    (token) => {
-      const totalLockedBalance = useMemo(
-        () => token.lockedBalances?.reduce((total, { amount }) => +amount + total, 0) ?? 0,
-        [token.lockedBalances]
-      );
+  const renderTokenCard = useCallback((token) => {
+    const totalLockedBalance = useMemo(
+      () => token.lockedBalances?.reduce((total, { amount }) => +amount + total, 0) ?? 0,
```
Comment (ManuGowda) [inline src/modules/wallet/components/overview/overview.js]:
```suggestion
      () => token.lockedBalances?.reduce((total, { amount }) => BigInt(amount) + BinInt(total), BigInt(0)).toString() ?? '0',
```

File: src/modules/wallet/components/TokenCard/TokenCard.js (hunk 1 above)
Comment (oskarleonard) [reply src/modules/wallet/components/TokenCard/TokenCard.js]: Fixed in [commit](https://github.com/LiskHQ/lisk-desktop/pull/4881/commits/0d76db36b68842c3bdb832d0bee5bd1f2dec14cc)

File: src/modules/wallet/components/overview/overview.js (hunk 2 above)
Comment (oskarleonard) [reply src/modules/wallet/components/overview/overview.js]: Fixed in [commit](https://github.com/LiskHQ/lisk-desktop/pull/4881/commits/0d76db36b68842c3bdb832d0bee5bd1f2dec14cc)

**Review by ManuGowda — APPROVED** [review]

**Review by eniolam1000752 — APPROVED** [review]
code base LGTM

**Pull request merged**